authpass/lib/ui/screens/entry_details.dart

```diff
@@ -1123,6 +1127,7 @@ class _EntryFieldState extends State<EntryField>
             _isValueObscured = true;
           }
         });
+        initController();
```

Comment:
I think we should move this one line up into the `setState()` callback. technically it makes no difference, because `setState` only marks the widget for being rebuild, but logically the `initController` changes the state, so we should probably do this inside the callback.

i think we can also make it private: `_initController`